### PR TITLE
Bhs/docker

### PR DESCRIPTION
**Pull request by bradwich**
Please review the workflow yaml (automake.yaml) and the README.md 

The source i used for the README.md file is [here](https://github.com/webmeshproj/webmesh/blob/main/README.md?plain=1)

### CODE REVIEW

File: .github/workflows/automake.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    branches:
+      - "*"
+    tags:
+      - v*
+jobs:
+  runs on: ubuntu-latest
+  steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+    - name: Run Unit Test
```
Comment (iansmith) [inline .github/workflows/automake.yaml:13]: Do you need a step here like:

```
- name: Build
  run: make
```

sorry if I didn't mention that before... you probably should do `make` before just to make sure it all builds

File: pullreq1.yml
Comment (iansmith) [inline pullreq1.yml:1]: What is this for? Is this different from the workflow(s) above?

File: test.yaml
Comment (iansmith) [inline test.yaml:1]: Again, what's this for?  I probably just don't understand GH actions well enough...

**Comment by bradwich** [comment]
Made a new file for testing therefore I'm closing this pr.

**Pull request closed without merging**